templates/sidebar.html

```diff
@@ -90,6 +90,31 @@
                             <p>Graph</p>
                         </a>
                     </li>
+                    <li class="nav-item">
+                        <a href="#" class="nav-link{% if '/quota-attribute/' in request.path or '/billing-group/' in request.path %} active{% endif %}">
+                            <i class="nav-icon fas fa-dollar-sign"></i>
+                            <p>
+                                Billing
+                                <i class="right fas fa-angle-down"></i>
+                            </p>
+                        </a>
+                        <ul class="nav nav-treeview">
+                            <li class="nav-item">
+                                <a href="{% url 'profiles:billing_group_list' %}"
+                                   class="nav-link {% if '/billing-group/' in request.path %}active{% endif %}">
+                                    <i class="nav-icon fas fa-users"></i>
+                                    <p>Groups</p>
+                                </a>
+                            </li>
+                            <li class="nav-item">
+                                <a href="{% url 'profiles:quota_attribute_definition_list' %}"
+                                   class="nav-link {% if '/quota-attribute/' in request.path %}active{% endif %}">
+                                    <i class="nav-icon far fa-chart-bar"></i>
+                                    <p>Quota</p>
+                                </a>
+                            </li>
```

Comment:
Place the quota block before groups as we need to configure this one first.
+ change icon for `<i class="fas fa-chart-pie"></i>`